gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -276,6 +312,14 @@ def configure_interfaces(self, context, resource_data):
             LOG.error(msg)
             raise Exception(msg)
 
+        try:
+            status = self._log_forwarding(mgmt_ip)
+        except Exception:
+            msg = ("Configuration of log forwarding has failed. Reason: %r" %
+                   status.json())
+            LOG.error(msg)
+            return status.json()
```

Comment:
Even after adding retries, there can be a case where this log forwarding can fail because of a persistent issue. So, if we fail the chain for log forwarding issue, it doesn't look good from the perspective of chain creation. From the perspectivie of visibility, it is a serious issue but, that is a bug we need to fix (just like VIP create failure in case of normal chain create).

One thing, we need to check is, why this API is failing. Once we know the nature of failure, we can take a decision on whether retries help or not.